docs/learn/calls.md

```diff
@@ -146,13 +153,43 @@ response = recommend_book("fantasy")
 print(response.content)
 ```
 
+In this example, we're setting the `temperature` parameter to 0.7, which affects the randomness of the model's output. A higher temperature (closer to 1.0) will result in more diverse and creative outputs, while a lower temperature (closer to 0.0) will make the outputs more focused and deterministic.
+
+### Dynamic Configuration
```

Comment:
should we add a blurb here mentioning the difference between using `BaseDynamicConfig` and provider specific versions like `openai.OpenAIDynamicConfig`?

seems worth mentioning that you can always use provider-agnostic configuration options (such as computed fields and metadata) but that you can also tie the prompt to a specific provider by using the provider specific config (and also then get provider specific type hints for things like call params but also messages etc.).

thoughts?